services/galley/src/Galley/API/Federation.hs

```diff
@@ -629,17 +627,19 @@ sendMLSCommitBundle remoteDomain msr = handleMLSMessageErrors $ do
   (ctype, qConvOrSub) <- getConvFromGroupId ibundle.groupId
   when (qUnqualified qConvOrSub /= msr.convOrSubId) $ throwS @'MLSGroupConversationMismatch
   -- this cannot throw the error since we always pass the sender which is qualified to be remote
-  Just resp <-
-    runErrorS @MLSLegalholdIncompatible $
-      postMLSCommitBundle
-        loc
-        (tUntagged @QRemote sender)
```

Comment:
this type application was placed here to prevent future changes from introducing errors. It is only safe to assume that we can discard the error when the sender is actually remote. since tUntagged works on local and remote, a future changed may go unchecked without this type application. 

also mind that while it's a minor thing, there's no need to convert `map` to `fmap`. it is more to type, more work for the specializer (longer compile times, possible missed specialisations and hence slower runtime, although the latter probably doesn't happen with `fmap`) and it obfuscates which type it is being applied to.

Reply:
@MangoIV stealing your comment and adding it to the code 👍🏼